project/Build.scala

```diff
@@ -41,13 +41,13 @@ object ScalikeJDBCProjects extends Build {
     incOptions := incOptions.value.withNameHashing(true),
     scalatestVersion := {
       CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, v)) if v <= 11 => "2.2.5"
-        case _ => "2.2.5-M2"
+        case Some((2, v)) if v <= 11 => "2.2.6"
+        case _ => "2.2.5-M3"
       }
     },
     specs2Version := {
       CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, v)) if v <= 11 => "2.4.17"
+        case Some((2, v)) if v <= 11 => "2.5"
         case _ => "3.6.3"
```

Comment:
http://repo1.maven.org/maven2/org/specs2/specs2-core_2.12.0-M3/

`"org.specs2" % "specs2-core_2.12.0-M3" % "3.6.3"` does not exists